ompi/mca/op/rocm/Makefile.am

```diff
@@ -0,0 +1,82 @@
+#
```

Comment:
High level comment to the components: since these op components have a direct depedencie to ROCm/CUDA, we need to make sure that they are *always* compiled as dso. I have not looked into this issue on how this was achieved for the other components (and hence I am not sure how to check for that right away), just want to add it as a reminder.

Reply:
This is the PR with the change for the opal accelerator components: https://github.com/open-mpi/ompi/pull/12055 I tried to find a similar mechanism in OMPI but no joy. Maybe @hppritcha knows where to mark components as DSO-by-default on the OMPI side?

Reply:
Added the `op/cuda` and `op/rocm` to `enable_mca_dso` in `opal_mca.m4`. That should do the trick (thanks @hppritcha)